test/test_files/terraformProject/main.tf

```
@@ -104,12 +104,12 @@ module "s3_bucket_remote" {
   }
 }
 
-module "remote_module_from_s3" {
-  source = "s3::https://s3.amazonaws.com/remote-module-cflint-prasheel/remote-module.zip"
-  tags = {
-    Owner     = "ankush.pandey@trilogy.com"
-  }
-}
+#module "remote_module_from_s3" {
+#  source = "s3::https://s3.amazonaws.com/remote-module-cflint-prasheel/remote-module.zip"
+#  tags = {
+#    Owner     = "ankush.pandey@trilogy.com"
+#  }
+#}
```

```suggestion
module "remote_module_from_s3" {
  source = "https://cloudfix-terraform-remote-module-demo.s3.amazonaws.com/remote-module.zip"
  tags = {
    Owner     = "ankush.pandey@trilogy.com"
  }
}
```
this should fix it, again ported from https://github.com/trilogy-group/cloudfix-aws/pull/3548/files#r1262364758